setup.md

```diff
@@ -113,10 +113,6 @@ There is an `./packages/nextjs/.env.sample` committed to the repo which contains
 
 :bulb: To get a list of scripts available, you can run `pnpm run` and it will output a list of available commands.
 
-- `local` - Runs Sanity Studio, Next.js app
-- `dev:nextjs` - Runs NextJs app
+- `dev:nextjs` - Runs the NextJs app
```

Comment:
⛏️ _(nit)_ The official docs seem to prefer `Next.js`.
```suggestion
- `dev:nextjs` - Runs the Next.js app
```

Reply:
Done